Gemfile

```diff
@@ -50,6 +50,7 @@ gem 'retries'
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
   gem "debug", platforms: %i[ mri mingw x64_mingw ]
+  gem 'simplecov', require: false
```

Comment:
I'm not sure that the coverage tool needs to available in development. The [documentation just has it in test](https://github.com/simplecov-ruby/simplecov#getting-started).